src/main/java/io/turntabl/employementprofilingsystem/Controllers/RequestController.java

```diff
@@ -0,0 +1,64 @@
+package io.turntabl.employementprofilingsystem.Controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.turntabl.employementprofilingsystem.Models.RequestTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@RestController
+public class RequestController {
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
```

Comment:
are you using a DAO for this? The controller uses the DAO, the DAO uses the JdbcTemplate. 

This is fine if you're not using a DAO...

Reply:
we're not using DAO